docs/socket-activation.rst

```diff
@@ -0,0 +1,47 @@
+Socket Activation
+-----------------
+
+While waitress does not support the various implementations of socket activation,
+for example using systemd or launchd, it is prepared to receive pre-bound applications
+from an init system.
```

Comment:
init system or other launchers that can provide pre-bound sockets. Chaussette for instance could also use this support, it's not strictly only init systems.